examples/client_realms/client_realms.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const mc = require('minecraft-protocol')
+
+const [,, username] = process.argv
+if (!username) {
+  console.log('Usage : node client_realms.js <username/email>')
+  process.exit(1)
+}
+
+const client = mc.createClient({
+  realms: {
+    // realmId: '1234567', // Connect the client to a Realm using the Realms ID
+    pickRealm: (realms) => realms.find(e => e.name === 'Realm Name') // Connect the client to a Realm using a function that returns a Realm
+  },
+  username: username,
+  auth: 'microsoft' // This option must be present and set to 'microsoft' to  join a Realm.
+})
+
+client.on('connect', function () {
+  console.info('connected')
+})
+client.on('disconnect', function (packet) {
+  console.log('disconnected: ' + packet.reason)
+})
+client.on('chat', function (packet) {
```

Comment:
this is outdated now, maybe add `realms` as an option to the existing chat example instead ? This would avoid needing to keep a separate chat example up to date (especially as 1.19.2 and 1.19.3 will change chat further)

Reply:
Ah ok, I think it'd be better to just remove the on chat event as the example's only really here to show the available options for connecting to a Realm. If we merge it with the current chat example it could get cumbersome with the host/port options etc.